wrappers/models_configurations/base_config.py

```diff
@@ -0,0 +1,24 @@
+
+from typing import Any
+from pathlib import Path
+from .config_types import ConfigTypes
+import json
+from abc import ABC, abstractmethod
+
+class BaseConfiguration(ABC):
+    def __init__(self, api_key: str) -> None:
+        self.api_key = api_key
+
+    def to_dict(self) -> dict[str, Any]:
+        return self.__dict__
+    
+    def to_file(self) -> Path:
+        config_path = Path(__file__).parent / self._get_config_file_name()
```

Comment:
This file is never being used. We can add "-l --load default=" command line arg, or come up with something better.

Reply:
Yeah that's a good plan, will tackle it in a separate PR